### PR TITLE
[ROS376] Create RBL tests from TransitionSpec.scala

### PR DESCRIPTION
**Pull request by AbnerZheng**
@tymm PTAL

### CODE REVIEW

**Comment by tymm** [comment]
@kjmcqueen can you also approve? You know more about testing the C++ version.

File: rosette/rbl/rosette/tests/run-tests.ros [hunk 1]
```diff
@@ -111,6 +111,10 @@
 (load "tests/consume-tests.rbl" 'silent)
 (load "tests/produce-tests.rbl" 'silent)
 
+;;; tests for transtion specification
+(load "tests/transition-spec.rbl"
```
Comment (AbnerZheng) [inline rosette/rbl/rosette/tests/run-tests.ros]: @kjmcqueen I added this file in here.

File: rosette/rbl/rosette/tests/transition-spec.rbl [hunk 2]
```diff
@@ -0,0 +1,50 @@
+(test-form (if #t 1 2) 1)
```
[the remark this answers is not in the document]
Comment (AbnerZheng) [reply rosette/rbl/rosette/tests/transition-spec.rbl]: I have added this file in `rosette/rbl/rosette/tests/run-tests.ros`,  you can see my comment below.

**Comment by AbnerZheng** [comment]
@tymm Could you please help me merge this PR?

**Pull request merged**